tensorrt_llm/serve/openai_server.py

```
@@ -975,6 +995,42 @@ async def create_stream_response(generator, request: ResponsesRequest, sampling_
+            "guided_decoder",
+            "spec_resource_manager",
+            "_no_capture_model_extra",
+            "executor_extra",
+            "kv_cache",
+            "model",
+            "draft_model"]
+        #self.llm._collective_rpc('sleep', args=(tags,))
+        print(f"HTTP received: release_memory {tags}")
+        return JSONResponse(content={"status": "success"})
+
+    async def resume_memory(self, request: MemoryUpdateRequest) -> JSONResponse:
```

Ruff (0.14.5)

Unused method argument: `request`

(ARG002)